charts/datadog-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: datadog-operator
-version: 1.0.5
-appVersion: 1.0.3
+version: 1.4.1
```

Comment:
This needs to be update in the `operator-eks-addon` chart and it's version bumped.